ipv6-lab.yml

```diff
@@ -0,0 +1,192 @@
+---
+- hosts: kube-nat64-dns64
+  tasks:
+    - name: update kernel
+      yum:
+        name: kernel
+        state: latest
+      notify: "restart server"
+  handlers:
+    - name: restart server
+      shell: sleep 2 && /sbin/shutdown -r now "System reboot"
+      async: 1
+      poll: 0
+      listen: "restart server"
+    - name: Wait for reboot
+      wait_for_connection:
+        delay: 5
+        timeout: 60
+      listen: "restart server"
+
+- hosts: all
+  tasks: []
+  roles:
+    - { role: ipv6-setup }
+
+- hosts: master,nodes
+  tasks:
+    - name: Remove default ipv4 route
+      shell: |
+        if ip route | grep default; then
+          ip route delete default via 192.168.122.1 dev eth0
+        fi
+
+- hosts: kube-master
+  tasks:
+    - name: configure routing
+      lineinfile:
+        path: /etc/sysconfig/network-scripts/route6-eth1
+        create: yes
+        line: "{{ item }}"
+      with_items:
+        - "64:ff9b::/96 via fd00::64 metric 1024"
+        - "fd00:101::/64 via fd00::101 metric 1024"
+        - "fd00:102::/64 via fd00::102 metric 1024"
+      notify: "restart eth1"
+  handlers:
+    - include: roles/ipv6-setup/handlers/main.yml
+      static: yes
+
+- hosts: kube-node-1
+  tasks:
+    - name: configure routing
+      lineinfile:
+        path: /etc/sysconfig/network-scripts/route6-eth1
+        create: yes
+        line: "{{ item }}"
+      with_items:
+        - "64:ff9b::/96 via fd00::64 metric 1024"
+        - "fd00:1234::/64 via fd00::100 metric 1024"
+        - "fd00:102::/64 via fd00::102 metric 1024"
+      notify: "restart eth1"
+  handlers:
+    - include: roles/ipv6-setup/handlers/main.yml
+      static: yes
+
+
+- hosts: kube-node-2
+  tasks:
+    - name: configure routing
+      lineinfile:
+        path: /etc/sysconfig/network-scripts/route6-eth1
+        create: yes
+        line: "{{ item }}"
+      with_items:
+        - "64:ff9b::/96 via fd00::64 metric 1024"
+        - "fd00:1234::/64 via fd00::100 metric 1024"
+        - "fd00:101::/64 via fd00::101 metric 1024"
+      notify: "restart eth1"
+  handlers:
+    - include: roles/ipv6-setup/handlers/main.yml
+      static: yes
+
+- hosts: kube-nat64-dns64
+  tasks:
+    - name: configure routing
+      lineinfile:
+        path: /etc/sysconfig/network-scripts/route6-eth1
+        create: yes
+        line: "{{ item }}"
+      with_items:
+        - "fd00:101::/64 via fd00::101 metric 1024"
+        - "fd00:102::/64 via fd00::102 metric 1024"
+      notify: "restart eth1"
+  handlers:
+    - include: roles/ipv6-setup/handlers/main.yml
+      static: yes
+
+- hosts: kube-nat64-dns64
+  tasks:
+    - name: install the 'Development tools' package group
+      yum:
+        name: "@Development tools"
+        state: latest
```

Comment:
latest is intended. I was getting some conflict with existing packages not being up to date.

Reply:
If you're installing the Development tools here fresh on a new VM, then use of `latest` shouldn't be necessary. This will fail ansible linting. If you really need to run an upgrade, then you should move `latest` into a package upgrade playbook, or make this a variable, with default to `present`.